src/directives/profile.js

```diff
@@ -12,9 +12,9 @@ goog.require('ngeo.profile');
  * Example:
  *
  *     <div ngeo-profile="ctrl.profileData"
- *          ngeo-profile-options="ctrl.profileOptions"
- *          ngeo-profile-pois="ctrl.profilePois"
- *     ></div>
+ *       ngeo-profile-options="ctrl.profileOptions"
+ *       ngeo-profile-pois="ctrl.profilePois">
+ *     </div>
```

Comment:
Should it has two more spaces ? (others examples seems to have 2 more space for indentation).